README.md

```diff
@@ -44,11 +74,10 @@ In node-mesh mode the nodes peers with all the nodes in order to exchange routes
 
 [kube-master]
 10.99.0.26
-
-[etcd]
-10.99.0.26
```

Comment:
Why not keeping a separate etcd cluster? it's not necessary bound to the master host